tests/lib/utilities/test_toUTC.py

```diff
@@ -4,10 +4,28 @@
 import sys
 import os
 sys.path.append(os.path.join(os.path.dirname(os.path.abspath(__file__)), '../../../lib'))
+import pytz
+
+import tzlocal
+
+
+def utc_timezone():
+    ''' This is a mock function, so when we run tests
+        we trick the system into thinking we're on UTC
+        no matter what the real timezone is
```

Comment:
Instead of mocking the method, we could just simply require this test to run on a system that is using UTC. This will however, fail on other systems, but maybe it gives us higher confidence that this determination of the timezone is working correctly.